README.md

```diff
@@ -1,9 +1,32 @@
-## Glitch Grid
+## Motivation
 
-This is a toy distributed system which we will use to demonstrate how to
-transform your system to run within [Antithesis](https://antithesis.com/). We have the same
-basic system implemented in Go and Python, as well as a language-agnostic
-workload which can be used to test the system.
+Glitch Grid is a toy distributed system for demonstrating how to
+interface your system with the [Antithesis Platform](https://antithesis.com/). This project demonstrates:
+
+* Use of an SDK for [assertion definition](https://antithesis.com/docs/using_antithesis/properties.html)
+   and control flow.
+* Build and, for Go, [source instrumentation](https://antithesis.com/docs/instrumentation/go_instrumentation.html)
```

Comment:
Maybe link to main instrumentation page instead of Go page? Not everybody looking at this project will be a Go person.